torch/_dynamo/polyfills/itertools.py

```diff
@@ -65,6 +71,18 @@ def islice(iterable: Iterable[_T], /, *args: int | None) -> Iterator[_T]:
                 next_i += step
 
 
+# Reference: https://docs.python.org/3/library/itertools.html#itertools.pairwise
+if sys.version_info >= (3, 10):
+    # itertools.pairwise was added at Python 3.10
+    @substitute_in_graph(itertools.pairwise, is_embedded_type=True)  # type: ignore[arg-type]
+    def pairwise(iterable: Iterable[_T], /) -> Iterator[tuple[_T, _T]]:
+        iterator = iter(iterable)
+        a = next(iterator, None)
+        for b in iterator:
+            yield a, b
+            a = b
```

Comment:
You may bypass the unsupported `next(iterator, None)` instruction by this:

```suggestion
    def pairwise(iterable: Iterable[_T], /) -> Iterator[tuple[_T, _T]]:
        a = None
        first = True
        for b in iterable:
            if first:
                first = False
            else:
                yield a, b
            a = b
```

It would be nice if you could open an issue to track the progress for `next(iterator, default)` support.